workflowai/core/domain/run.py

```diff
@@ -86,6 +87,47 @@ def model(self):
             return None
         return self.version.properties.model
 
+    def format_output(self) -> str:
+        """Format the run output as a string.
+
+        Returns a formatted string containing:
+        1. The output as a nicely formatted JSON object
+        2. The cost with $ prefix (if available)
+        3. The latency with 2 decimal places and 's' suffix (if available)
+
+        Example:
+            Output:
+            ==================================================
+            {
+              "message": "hello"
+            }
+            ==================================================
+            Cost: $ 0.001
+            Latency: 1.23s
+        """
+        # Create a dictionary with the output data
+        output_data = self.output.model_dump()
```

Comment:
```suggestion
        output_data = self.output.model_dump_json(indent=2)
```

This way you don't have to dump it below